projects/novo-elements/src/utils/form-utils/FormUtils.ts

```diff
@@ -140,9 +140,17 @@ export class FormUtils {
     };
     if (field.type === 'TO_MANY') {
       if (field.associatedEntity && ~this.ASSOCIATED_ENTITY_LIST.indexOf(field.associatedEntity.entity)) {
-        type = 'entitychips'; // TODO!
+        if (field.multiValue === false) {
+          type = 'entitypicker';
+        } else {
+          type = 'entitychips';
+        }
       } else {
-        type = 'chips';
+        if (field.multiValue === false) {
+          type = 'picker';
+        } else {
+          type = 'chips';
+        }
```

Comment:
please update unit tests to reflect this change

Reply:
Definitely!